metabase/metabase
# hide dashcard when empty setting

Related to https://github.com/metabase/metabase/issues/29175

### Description

Adds a new dashcard setting that defines if the card should be visible on a dashboard when results are empty.

### How to verify

1. Create a new dashboard with a few cards
2. Open the dashcards settings and toggle they should be hidden when no results
3. Connect filters to the dashcards
4. Enter filter values that lead to empty results
5. Ensure cards get hidden

### Demo

https://metaboat.slack.com/archives/C04D0E4837E/p1683940774587059

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30630)
